### ISSUE TITLE
Ability to contact the owner of the item to ask questions

### ISSUE THREAD

**Issue by danaebio** [robeastwood/repaircafe.leightonbuzzard.org#33]
Or ability to put comments on the item to be able to ask questions (and the person that registered the item gets an email notification when a comment has been made)

**Comment by robeastwood** [comment]
Yep good plan! The ability to add notes (from both sides) is already there, but notifications would be helpful im sure

**Comment by danaebio** [comment]
Thank you
